github-actions/kind/action.yml

```diff
@@ -34,6 +38,15 @@ runs:
         echo "Container image registry started at ${REGISTRY_ADDRESS}"
 
         KIND_CONFIG_FILE=${{ env.TEMP_DIR }}/kind.yaml
+
+        WORKER_NODES=${{ inputs.worker-nodes }}
+
+        if [ "$WORKER_NODES" -gt 0 ]; then
+          for i in $(seq 1 $WORKER_NODES); do
+            sed -i "/^nodes:/a \ \ - role: worker\n    image: kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1" ${KIND_CONFIG_FILE}
```

Comment:
Can you also add labels for worker nodes - https://kind.sigs.k8s.io/docs/user/configuration/#extra-labels?
You can i.e. provide label prefix as action parameter.